helpers/collaborators.ts

```diff
@@ -10,7 +10,8 @@ const getCollaborators = async ({
   octokit: Octokit;
   owner: string;
   repo: string;
-}): Promise<Array<Collaborator>> => {
+}): Promise<Collaborators> => {
+  let allCollaborators: Collaborators = {};
```

Comment:
_Nit:_ Can you move this closer to where it's being used? But I know some people like top level declares so 🤷‍♂

Reply:
_Nit:_ Also slightly prefer `collaborators` over `allCollaborators`